### PR TITLE
Feature/midl tabs

### PR DESCRIPTION
**Pull request by pmandler**

### CODE REVIEW

File: venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   });
+
+  Webfield.ui.spinner('#notes');
+}
+
+function renderConferenceTabs() {
+  var sections = [
+    {
+      heading: 'Full - Accept(Oral)',
```
Comment (zbialecki) [inline venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: Should there be a space between Accept and (Oral)?

File: venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      id: 'accepted-oral-papers',
+    },
+    {
+      heading: 'Full - Accept(Poster)',
```
Comment (zbialecki) [inline venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: Also here?

File: venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+  var paperDisplayOptions = {
+    pdfLink: true,
+    replyCount: true,
+    showContents: true
+  };
+
+  Webfield.ui.searchResults(
+    oralDecisions,
+    _.assign({}, paperDisplayOptions, {showTags: false, container: '#accepted-oral-papers'})
```
Comment (zbialecki) [inline venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: I would move `showTags: false` to the paperDisplayOptions so it doesn't need to be repeated for each tab.

Comment (pmandler) [reply venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: Not sure what to do about slow rendering - do you have a suggestion?

**Comment by pmandler** [comment]
Add consoles tab.

File: venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js (hunk 3 above)
Comment (zbialecki) [reply venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: How many total submissions are there for this conference?

Comment (zbialecki) [reply venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: It's probably not an issues unless there are greater than 1000 submissions.

Comment (pmandler) [reply venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: There are under 200.

Comment (zbialecki) [reply venues/MIDL.io/2019/Conference/webfield/conferenceWebfield_decision_tabs.js]: Oh this is fine then.

**Comment by melisabok** [comment]
is this ready to merge?

**Comment by pmandler** [comment]
Yes please

**Pull request merged**